README.md

```diff
@@ -10,11 +14,19 @@ git remote set-url origin YOUR_NEW_REPO_URL_HERE
 git branch -M main
 git push -u origin main
 
+To install dependencies, seed local database, and run tests look at package.json to find the scripts,
+for the dependencies input into terminal "npm install ${dependency name}"
```

Comment:
You can just run 'npm install' and it will install all packages listed in the json